### DISCUSSION TITLE
How can I disable netdata to add all apps into apps_groups_root_target?

### DISCUSSION THREAD

**Discussion by chenxiangw** [netdata/netdata#18739] [Q&A]
For old version of netdata, if I specify some group in apps_groups.conf, only the specific group matrix will be saved in database. But for latest version, netdata will add all apps into targets including those I didn't add in apps_groups.conf.
<img width="696" alt="image" src="https://github.com/user-attachments/assets/c5cc248e-3512-4f9f-849c-ef995e7f60b6">
As shown in log, I didn't add **colord** in apps_groups.conf but it was added into target, and as a result many unexpected app matrixes are present in the chart.
How can I disable those unexpected app matrixes?

**Comment by ilyam8** [comment]
Add `managers: clear` to your config. All other processes will be aggregated into pid 1 manager (e.g. `systemd`).

**Reply by chenxiangw** [reply]
It doesn't work for me.

**Reply by ilyam8** [reply]
Works for me.
